docs/03-tutorials/00-api-exposure/apix-03-expose-and-secure-service.md

```diff
@@ -157,22 +192,57 @@ The exposed service requires tokens with "read" scope for `GET` requests in the
   Secure a Function
   </summary>
 
-1. Create a Function using the [supplied code](./assets/function.yaml):
+1. Create a Function in your Namespace using the [supplied code](./assets/function.yaml):
```

Comment:
When I clicked the link, I was surprised that it's a download and not just a link. Can we point out that clicking the link immediately starts a download?

Something like that:

```suggestion
1. Create a Function in your Namespace using the supplied code [(download function.yaml)](./assets/function.yaml):
```